src/js/addRemove.js

```diff
@@ -0,0 +1,36 @@
+export const add = (list) => {
+  const inputField = document.querySelector('.input-container .task-input');
+  const spanalert = document.querySelector('.input-container .alert');
+  let count = localStorage.getItem('count') && list.length ? localStorage.getItem('count') : 1;
```

Comment:
- [ ] This statement makes your indexes not correct. They are actually `strings` instead of `numbers` after the first task is created. This issue compromise all your functionalities. You don't necessarily need a count variable here but you if want you can set it to the length of your list. Please try to fix it and you will be able to fix the other ones.